pkg/infra/baseAgent.go

```diff
@@ -210,10 +233,12 @@ func (a *Base) TakeFood(amountOfFood food.FoodType) (food.FoodType, error) {
 	if amountOfFood < 0 {
 		return 0, &NegFoodError{}
 	}
+	a.totalFoodSeen += a.tower.currPlatFood
```

Comment:
This is called:
- Once per tick
- Only if agent is calling `TakeFood`

Which means that if the agent never tried to take food we are not logging that they are seeing this food

Reply:
I've moved this into a function that gets called on each tick and occurs before each agent takes action